code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -125,6 +125,19 @@
 	origin_tech = "combat=4;magnets=4;powerstorage=3"
 	shaded_charge = 1
 
+/obj/item/gun/energy/immolator/multi
+	name = "multi lens immolator cannon"
+	desc = "A large laser cannon, similar to the Immolator Laser, with togglable firemodes. It is frequently used by military-like forces through Nanotrasen."

```

Comment:
`toggleable`

Its one that gets argued over a lot, and hasn't been 'officially' called upon by Webster or the like because its not considered a word yet in English. However, similar words that follow three consonants keep the E.

Ignore or not, up to you - just an interesting syntactic point for the English nerds.

Reply:
`toggleable` feels better.